README.md

```diff
@@ -5,3 +5,313 @@ qgepplugin
 This plugin is part of the QGEP Wastewater project for QGIS.
 
 [![Build Status](https://travis-ci.org/QGEP/qgepplugin.svg?branch=master)](https://travis-ci.org/QGEP/qgepplugin)
+
+# QGEP-SWMM
+
+## Installation
+
+### Install QGEP plugin 
+
+* Download the plugin: https://github.com/iggroup/qgepplugin/tree/swmm/src
+* Save in the QGIS plugin path in a folder named `qgepplugin`.
+
+### SWMM Installation
+* Download and install [SWMM](https://www.epa.gov/water-research/storm-water-management-model-swmm)
+
+* In QGIS `Preferences > Processing > Processing providers > QGEP > SWMM exectutable` set the path to the swmm `.exe` i.e. `C:/Program Files (x86)/EPA SWMM 5.1.013/swmm5.exe` 
+
+### Installation psycopg2
+The script requires `psycopg2` to connect to the database. It is installed by default with QGIS, it is not necessary to install it if you use the QGEP plugin.
+
+[Psycopg2 installation](https://pypi.org/project/psycopg2/)
+
+### Installation of a postgreSQL service
+
+In `pg_service.conf` create a service to connect to the database that you want to analyse. You can also use an existing service.
+
+```
+[pg_qgep_demo_data]
```

Comment:
Can we stick to `pg_qgep` as we do in the documentation too? I think that will be less confusing.
Also this whole part might be deleted from here and linked to the main documentation to avoid duplication.

Reply:
I deleted this part and made a link to the main doc.